frontend/src/components/molecules/SustainabilityGoalsList.tsx

```diff
@@ -0,0 +1,57 @@
+import { Center, SimpleGrid } from '@chakra-ui/react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getSustainabilityGoals } from '../../api/ontologies';
+import { mapIdToNode } from '../../common/node';
+import { setError } from '../../state/reducers/apiErrorReducer';
+import { selectNode } from '../../state/reducers/ontologyReducer';
+import { Node, SustainabilityGoal } from '../../types/ontologyTypes';
+import IconContainer from '../atoms/IconContainer';
+
+// Initial node displayed on the frontpage
+const sustainabilityNode: Node = {
```

Comment:
This shouldn't be necessary. There's no reason for getSustainabilityGoals to need a node ID as argument.